spec/requests/users_spec.rb

```diff
@@ -2,6 +2,19 @@
 
 RSpec.describe 'Users', type: :request do
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    it 'returns a successful response' do
+      get '/users'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the index template' do
+      get '/users'
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get '/users'
+      expect(response.body).to include('Placeholder text for all users')
+    end
   end
 end

```

Comment:
- Also, according to the project requirements you are required to test for all actions in the controller, I suggest you also test for the `User#show` action. Please fix this 🙏